src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -177,5 +209,338 @@ private void ThrowIfDisposed()
                 throw new ObjectDisposedException(null);
             }
         }
+
+        private void InitializeMetricsClients()

```

Comment:
Using this as a way to avoid spending the resources creating these clients if the listener isn't going to be using runtime scale.

Reply:
How about just using properties with lazy init?